pandas/tests/series/test_missing.py

```diff
@@ -1317,3 +1317,38 @@ def test_series_interpolate_intraday(self):
         result = ts.reindex(new_index).interpolate(method='time')
 
         tm.assert_numpy_array_equal(result.values, exp.values)
+
+    # TODO: De-duplicate with similar tests in test.frame.test_missing?
```

Comment:
this is much more complicated we have generic tests that already do a lot of this

you can also move these tests there

Reply:
I'll take a look for a better place.

This is also the answer to the previous question about why not starting with DataFrame: bc the test originally tested Series and DataFrame in the same test.

Reply:
> this is much more complicated we have generic tests that already do a lot of this

Both for this PR and the upcoming slew of Datetime/Timedelta/PeriodArray tests, there are a bunch of for FooArray/FooIndex/Series[foo]/DataFrame[foo-column] that are highly duplicative and I think things are falling through the cracks.  Think we can find a place to put them so they can be parametrized?